src/components/SecondarySlider/SecondarySlider.tsx

```diff
@@ -143,13 +146,17 @@ export const SecondarySlider: FC<Props> = ({ endpoint, title }) => {
                 key={product.id}
                 className="secondary-slider__card"
               >
+<<<<<<< HEAD
+                {product.name}
+=======
```

Comment:
```suggestion
                {product.name}
```
remove unneсessary strings)